types/primer__octicons/index.d.ts

```diff
@@ -0,0 +1,333 @@
+// Type definitions for @primer/octicons 16.0
+// Project: https://github.com/primer/octicons
+// Definitions by: Bread <https://github.com/vBread>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace octicons {
+    interface Icon {
+        /**
+         * Returns the string of the symbol name, same as the key for that icon.
+         */
+        readonly symbol: string;
+
+        /**
+         * Returns an array of keywords for the icon.
+         */
+        readonly keywords: string[];
+
+        /**
+         * The `heights` property allows you to access all the SVGs for an icon
+         * using the natural height of the SVG.
+         */
+        readonly heights: IconHeights;
+
+        /**
+         * Returns a string of the `<svg>` tag.
+         *
+         * @param options An optional object used to add CSS class names,
+         * accessibility options, and sizes.
+         */
+        toSVG(options?: SVGOptions): string;
+    }
+
+    interface IconHeights {
+        readonly 16: IconHeight<16>;
+        readonly 24?: IconHeight<24>;
+    }
+
+    interface IconHeight<N extends 16 | 24> {
+        /**
+         * Returns the icon's true width, based on the SVG view box width.
+         *
+         * Note, this doesn't change if you scale it up with size options, it
+         * only is the natural width of the icon.
+         */
+        readonly width: N;
+
+        /**
+         * Returns the string representation of the path of the icon.
+         */
+        readonly path: string;
+
+        /**
+         * This is an object of all the attributes that will be added to the output tag.
+         */
+        readonly options: IconAttributes;
+    }
+
+    interface IconAttributes {
+        version: string;
+        width: string;
+        height: string;
```

Comment:
```suggestion
        height: number;
```
https://github.com/primer/octicons/blob/main/lib/octicons_node/index.js#L55
(assuming is succeds)